README.md

```diff
@@ -1,44 +1,250 @@
+
+
+```
+
+### zero apply
+
+The `zero apply` command takes the templated modules generated based on your input and spins up a scalable & performant infrastructure for you!
+
+```shell
+$ zero apply
+
+# Sample Output
+Choose the environments to apply. This will create infrastructure, CI pipelines, etc.
+At this point, real things will be generated that may cost money!
+Only a single environment may be suitable for an initial test, but for a real system we suggest setting up both staging and production environments.
+✔ Production
+🎉  Bootstrapping project zero-init. Please use the zero-project.yml file to modify the project as needed.
+Cloud provider: AWS
+Runtime platform: Kubernetes
+Infrastructure executor: Terraform
+
+...
+...
+
+
+✅  Done.
+Your projects and infrastructure have been successfully created.  Here are some useful links and commands to get you started:
+zero-aws-eks-stack:
+- Repository URL: github.com/myapp-org/infrastructure
+- To see your kubernetes clusters, run: 'kubectl config get-contexts'
+- To switch to a cluster, use the following commands:
+- for production use: kubectl config use-context arn:aws:eks:us-west-2:123456789:cluster/myapp-infra-production-us-west-2
+
+- To inspect the selected cluster, run 'kubectl get node,service,deployment,pods'
+zero-deployable-react-frontend:
+- Repository URL: github.com/myapp-org/frontend
+- Deployment Pipeline URL: https://app.circleci.com/pipelines/github/myapp-org/frontend
+- Production Landing Page: app.commitzero.com
+
+zero-deployable-backend:
+- Repository URL: github.com/myapp-org/backend-service
+- Deployment Pipeline URL: https://app.circleci.com/pipelines/github/myapp-org/backend-service
+- Production API: api.commitzero.com
+```
+
+***Your stack is now up and running, follow the links your terminal to visit your application 🎉*** 
+
+
+## Zeros Default Stack
+
+![systerm-architecture](https://raw.githubusercontent.com/commitdev/zero-aws-eks-stack/master/templates/docs/architecture-overview.svg)
+
+If you would like to learn more about the zero-aws-eks stack, you can read more about it [here](https://github.com/commitdev/zero-aws-eks-stack)
+___
+
+## Contributing to Zero 
+
+Zero welcomes collaboration from the community; you can open new issues in our GitHub repo, Submit PRs' for bug fixes or browse through the tickets currently open to see what you can contribute too.
+
+### Building this tool
 
 ```shell
 $ git clone git@github.com:commitdev/zero.git
 $ cd zero && make
 ```
-#### Running the tool locally
+
+### Running the tool locally
 
 To install the CLI into your GOPATH and test it, run:
-```
+
+```shell
 $ make install-go
 $ zero --help
 ```
+___
+## FAQ
+
+Why is my deployed application not yet accesible?
 
+- It takes about 20 - 35 mins for your deployed application to be globally available through AWS CloudFront CDN.
 
+___
 
-## Planning and Process
+## Learn More about Zero
```

Comment:
This is only internal to us for now, maybe call it Resources (since many of these links public won't have access to)